.github/workflows/integrate.yaml

```diff
@@ -29,29 +29,29 @@ jobs:
         with:
           php-version: "${{ matrix.php-version }}"
 
-      - name: "Set up Java"
-        uses: "actions/setup-java@v3"
-        with:
-          distribution: "zulu"
-          java-version: "11"
-
       - name: "Install dependencies with composer"
         run: "composer install --no-interaction --optimize-autoloader --prefer-dist"
 
       - name: "Install phive"
-        run: "ant getphive"
+        run: |
+          wget https://phar.io/releases/phive.phar
+          wget https://phar.io/releases/phive.phar.asc
+          gpg gpg-get-key --recv-keys 0x9D8A98B29B2D5D79
+          gpg --verify phive.phar.asc phive.phar
+          chmod +x phive.phar
+          rm phive.phar.asc
 
       - name: "Install dependencies with phive"
-        run: "ant -Dphive.bin=./phive.phar install-tools"
+        run: "phive install --copy --trust-gpg-keys 4AA394086372C20A,2A8299CE842DD38C,CF1A108D0E7AE720,2DE50EB60C013FFFA831040D12CE0F1D262429A5,E82B2FB314E9906E"
```

Comment:
How does this call succeed? The download above stores it as `phive.phar`. Either it's wrong or the wget lines are useless ;)